src/locales/en/translation.json

```diff
@@ -125,6 +125,9 @@
   "inviteUser": "Invite user",
   "invoiceNo": "Invoice #",
   "invoices": "Invoices",
+  "koatuuFirstLevel": "First level of subordination",
+  "koatuuSecondLevel": "Second level of subordination",
+  "koatuuThirdLevel": "Third level of subordination",
```

Comment:
```suggestion
  "firstLevelOfSubordination": "First level of subordination",
  "secondLevelOfSubordination": "Second level of subordination",
  "thirdLevelOfSubordination": "Third level of subordination",
```

Reply:
Thank you for the suggestion! I have implemented the changes.